README.md

```diff
@@ -106,11 +74,45 @@ pytest
 # Build and install
 python setup.py install
 ```
-### Conda install
-You can conda install CLX on an existing RAPIDS container. A RAPIDS image suitable for your environment can be pulled from https://hub.docker.com/r/rapidsai/rapidsai/. 
+### Conda Install 
+You can conda install CLX on an existing RAPIDS container. A RAPIDS image suitable for your environment can be pulled from [https://hub.docker.com/r/rapidsai/rapidsai/](https://hub.docker.com/r/rapidsai/rapidsai/). 
+
 ```
 conda install -c rapidsai-nightly -c rapidsai -c nvidia -c pytorch -c conda-forge -c defaults clx
 ```
+## Example Notebooks
+The notebooks folder contains example use cases and workflow instantiations.
+
+## Getting Started
+
+CLX is targeted towards cybersecurity data scientists, senior security analysts, threat hunters, and forensic investigators. Data scientists can use CLX in traditional Python files and Jupyter notebooks. CLX also includes structure in the form of a workflow. A workflow is a series of data transformations performed on a [GPU dataframe](https://github.com/rapidsai/cudf) that contains raw cyber data, with the goal of surfacing meaningful cyber analytical output. Multiple I/O methods are available, including Kafka and on-disk file stores.
+
+Example flow workflow reading and writing to file:
+
+```python
+from clx.workflow import netflow_workflow
```

Comment:
A small note to consider moving code snippet closer to the top of the README and look into using a different code snippet example if needed

Reply:
Good idea. I'll come up with a different code snippet and move this to the top to retain consistency across RAPIDS.

Reply:
Moving this out to different issue for 0.13